substrate/frame/revive/rpc/src/lib.rs

```diff
@@ -145,9 +145,10 @@ impl EthRpcServer for EthRpcServerImpl {
 		block: Option<BlockNumberOrTag>,
 	) -> RpcResult<U256> {
 		log::trace!(target: LOG_TARGET, "estimate_gas transaction={transaction:?} block={block:?}");
-		let hash = self.client.block_hash_for_tag(block.unwrap_or_default().into()).await?;
+		let block = block.unwrap_or(BlockNumberOrTag::BlockTag(BlockTag::Pending));
```

Comment:
is that the default when you call eth_estimateGas on Geth as well?

Reply:
according to my test it's using latest, tools like cast will also specify latest